cosmos/titan-test.json

```diff
@@ -0,0 +1,53 @@
+{
+  "bech32Config": {
+    "bech32PrefixAccAddr": "titan",
+    "bech32PrefixAccPub": "titanpub",
+    "bech32PrefixConsAddr": "titanvalcons",
+    "bech32PrefixConsPub": "titanvalconspub",
+    "bech32PrefixValAddr": "titanvaloper",
+    "bech32PrefixValPub": "titanvaloperpub"
+  },
+  "bip44": {
+    "coinType": 118
+  },
+  "chainId": "titan-test-1",
+  "chainName": "titan",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/titan-test/chain.png",
+  "currencies": [
+    {
+      "coinDecimals": 6,
+      "coinDenom": "TTN",
+      "coinGeckoId": "titan",
+      "coinMinimalDenom": "uttn",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/titan-test/uttn.png"
+    }
+  ],
+  "features": [],
+  "feeCurrencies": [
+    {
+      "coinDecimals": 6,
+      "coinDenom": "TTN",
+      "coinGeckoId": "titan",
+      "coinMinimalDenom": "uttn",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/titan-test/uttn.png",
+      "gasPriceStep": {
+        "average": 0.025,
+        "high": 0.03,
+        "low": 0.005
+      }
+    }
+  ],
+  "rest": "http://lcd.titannet.io:1317",
+  "rpc": "http://rpc.titannet.io:26657",
+  "stakeCurrency": {
+    "coinDecimals": 6,
+    "coinDenom": "TTN",
+    "coinGeckoId": "titan",
```

Comment:
This ID is currently unavailable now on CoinGecko. Please try again later when it becomes accessible. Alternatively, consider removing it, if you want this to be pushed promptly.

Reply:
Do you mean "titan"? I searched for "TTN" on CoinGecko and there are similar ones. Can I still use it?

Reply:
coinGeckoId you provided is unable to access.
https://www.coingecko.com/en/coins/titan

You should remove it if it's not a officially registered coinGeckoId.

Reply:
very thankful